helper-cli/src/main/kotlin/commands/packageconfig/RemoveEntriesCommand.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.packageconfig
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import org.ossreviewtoolkit.helper.common.writeAsYaml
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.ScanResult
+import org.ossreviewtoolkit.model.config.PackageConfiguration
+import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.model.utils.FindingCurationMatcher
+import org.ossreviewtoolkit.utils.expandTilde
+
+internal class RemoveEntriesCommand : CliktCommand(
+    help = "Removes all non-matching path excludes and license finding curations."
+) {
+    private val packageConfigurationFile by option(
+        "--package-configuration-file",
+        help = "The package configuration."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = true, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val ortResultFile by option(
+        "--ort-result-file",
+        help = "The ORT result file to read as input which should contain an scan result to which the given " +
+                "package configuration configuration applies to."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    override fun run() {
+        val packageConfiguration = packageConfigurationFile.readValue<PackageConfiguration>()
+        val ortResult = ortResultFile.readValue<OrtResult>()
+        val scanResult = ortResult.getScanResultFor(packageConfiguration)
```

Comment:
Consider aborting and printing and error here if the `scanResult` is null. Otherwise it might be confusing why this command does not work.

Reply:
Done.